mslib/msui/qt5/ui_linearview_window.py

```diff
@@ -9,6 +9,7 @@
 
 from PyQt5 import QtCore, QtGui, QtWidgets
 
+import mslib.msui.mpl_qtwidget
```

Comment:
this might be gone when recreated by pyuic5, or?

Reply:
correct.
Here, I moved the from x import y statement from below to this above to avoid  problems w.r.t. in circular imports.